diff --git a/out/util/electronGet.js b/out/util/electronGet.js
index 8a385317111f4d1e0fcd583a34d194c44fc50771..4611441db9cc983c217c3926d4dbdc553a47bb13 100644
--- a/out/util/electronGet.js
+++ b/out/util/electronGet.js
@@ -7,7 +7,7 @@ exports.downloadElectronArtifactZip = downloadElectronArtifactZip;
 exports.downloadElectronArtifact = downloadElectronArtifact;
 exports.getBinariesMirrorUrl = getBinariesMirrorUrl;
 const get = require("@electron/get");
-const get_1 = require("@electron/get");
+const get_1 = { ElectronDownloadCacheMode: (() => { const e = {}; e[e["ReadWrite"] = 0] = "ReadWrite"; e[e["ReadOnly"] = 1] = "ReadOnly"; e[e["WriteOnly"] = 2] = "WriteOnly"; e[e["Bypass"] = 3] = "Bypass"; return e; })() };
 const builder_util_1 = require("builder-util");
 const multiProgress_1 = require("electron-publish/out/multiProgress");
 const fs_1 = require("fs");
